frontend/src/lib/hooks/useGlobalCurrency/index.ts

```diff
@@ -0,0 +1,22 @@
+import { Fiat } from '../../../model/fiat';
+import {
+  currencies,
+  currencyToSymbolMap,
+} from '../../../pages/vaultSettings/vaultCurrency/constants';
+import {
+  PersistentStateKey,
+  usePersistentState,
+} from '../../../state/persistentState';
+
+export const useGlobalCurrency = () => {
+  const [globalCurrency, setGlobalCurrency] = usePersistentState(
+    PersistentStateKey.Currency,
+    currencies[0]
+  );
+
+  return {
```

Comment:
`export const useGlobalCurrency = () => {
  return usePersistentState(
    PersistentStateKey.Currency,
    currencies[0]
  );
  }`
  
 No need to return an object here. If you want - you can have a separate hook for a symbol. 
 
 `changeGlobalCurrency: (newCurrency: Fiat) => setGlobalCurrency(newCurrency)`  - no need in this.